packages/extension-chakra-storefront/src/setup-app.tsx

```diff
@@ -127,3 +120,18 @@ class ChakraStorefront extends ApplicationExtension<Config> {
 }
 
 export default ChakraStorefront
+
+// Flatten routes with multiple paths
+const flattenRoutes = (routes: RouteProps[]): RouteProps[] => {
+    const flatten: RouteProps[] = []
+    routes.forEach((route) => {
+        if (Array.isArray(route.path)) {
+            route.path.forEach((path) => {
+                flatten.push({path, component: route.component})
+            })
+        } else {
+            flatten.push(route)
+        }
+    })
+    return flatten
+}
```

Comment:
Do we have to flatten this?

Reply:
Yes, we do. Our `configureRoutes` expects/assumes it. Previously, we haven't had multiple paths in array: https://github.com/SalesforceCommerceCloud/pwa-kit/blob/92fb26a02bd13789eaf9a7d3855edb5c894ebdfa/packages/template-retail-react-app/app/routes.jsx#L99-L106

Reply:
It sounds like we are adding a fix for a problem that already exists outside of extensibility.. e.g. the `configureRoutes` util isn't compatible fully with the route config object. Would it not make more sends to fix this in the configureRoutes util?